scm/client.go

```diff
@@ -71,8 +71,21 @@ type (
 	// BranchListOptions specifies optional branch search term and pagination
 	// parameters.
 	BranchListOptions struct {
-		SearchTerm      string
-		PageListOptions ListOptions
+		ListOptions
```

Comment:
Changing this struct would be a backward incompatible change.

Reply:
agreed, have reverted this